opds.py

```diff
@@ -156,7 +156,7 @@ def library_catalog(
                                     href=library.web_url,
                                     type="text/html")
 
-        if library.logo and include_logo:
+        if include_logo and library.logo:
```

Comment:
Without `include_logo` to short-circuit this code, we'll refer to `library.logo` for every library and fetch it from the database anyway. It will actually still be faster than retrieving it as part of the big query, but we don't need it at all right now.